### ISSUE TITLE
minimap waveform only appears after resizing the window 

### ISSUE THREAD

**Issue by piem** [katspaugh/wavesurfer.js#954]
hi all, and thank you for the nice work.

I have an issue with minimap, where the waveform only appear after resizing the window. This issue existed in verions before 1.3.0, so I guess it's not related to #951. I still have to find a minimal test case to show it, since the examples work as expected.

cheers, piem

**Comment by entonbiba** [comment]
@piem are you calling `minimap1.render();`? 

I made it work with the following:
````
var minimap1;
wavesurfer1.on('ready', function () {
         minimap1 = null;
         var minimap1 = wavesurfer1.initMinimap({
             container: document.querySelector('#miniwavecontainer'),
             height: 30,
             waveColor: '#ddd',
             progressColor: '#999',
             cursorColor: '#68A93D'
         });
         minimap1.render();
});
````

**Comment by piem** [comment]
hi @entonbiba ,

No, this doesn't seem to be enough. I applied #955, but I'm still seeing this behaviour. It seems that the render function is being called, but too early.
 
Also, the container parameter has no effect, and the minimap is always created inside the main waveform container.

best, piem

**Comment by entonbiba** [comment]
@ may you provide the code you are using? The sample below works fine on multiple browsers on my end.

http://codepen.io/entonbiba/pen/ggeGyG

**Comment by entonbiba** [comment]
@piem I'm closing this since it's fixed in the latest version.

Here is a working example.
http://codepen.io/entonbiba/pen/ggeGyG

**Comment by piem** [comment]
hi!

yes, i confirm this is fixed in 1.3.1. Thanks for your work!

best, piem
